app/src/services/api.ts

```diff
@@ -1141,6 +1145,37 @@ export const api = createApi({
         }),
         transformResponse: (response: { message: string }) => response,
       }),
+      migrateHiapSelections: builder.mutation({
+        query: (data: { projectId: string; year: number }) => ({
+          url: `/admin/project/${data.projectId}/migrate-hiap-selections`,
+          method: "POST",
+          body: { year: data.year },
+        }),
+        transformResponse: (response: {
+          message: string;
+          citiesProcessed: number;
+        }) => response,
+      }),
+      startBulkHiapPrioritization: builder.mutation({
+        query: (data: {
+          projectId: string;
+          year: number;
+          actionType: ACTION_TYPES;
+          language?: string;
+        }) => ({
+          url: `/admin/bulk-hiap-prioritization`,
```

Comment:
Does it make sense to have the hiap routes in the admin path if we plan to enable self-service HIAP in the future? Or will we make a new route to accomodate this then?